README.md

```diff
@@ -81,9 +81,12 @@ class RateLimiterStore implements Store
         return Cache::get('rate-limiter', []);
     }
 
-    public function push(int $timestamp, int $limit)
+    public function push(int $timestamp, int $limit): void
     {
-        Cache::put('rate-limiter', array_merge($this->get(), [$timestamp]));
+        $current = array_slice($this->get(), -($limit + 1));
```

Comment:
Put `-($limit + 1))` in a variable with a meaningful name and use that variable.